common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_1905_vs.yaml

```diff
@@ -0,0 +1,37 @@
+#
+---
+_include: {
+  beerocks/bcl/beerocks_message_structs.h,
+  beerocks/tlvf/beerocks_message_action.h,
+}
+
+_namespace: beerocks_message
+
+_multi_class: True
+_multi_class_auto_insert:
+  action_op:
+    _type: eActionOp_1905_VS
+    _value_const: ACTION_TLV_VENDOR_SPECIFIC  
```

Comment:
There is end-of-line whitespace here, and also a few lines below.

However, it doesn't make much sense to fix this unless it can be captured automatically. I'll add yamllint to CI (at some unspecified time in the future :-) and fix it up then.

Reply:
OK, not fixing

Reply:
> BTW, how can you see spaces in Github?

I can't. I do most of the review based on `git log -p`, or sometimes I even do a `git rebase -x false prpl/master` to examine and test each commit individually.